Documentation/gettingstarted/local-redirect-policy.rst

```diff
@@ -0,0 +1,233 @@
+    01:36:24.608638 IP 10.16.215.55.60876 > 10.16.70.187.80: Flags [.], ack 1, win 221, options [nop,nop,TS val 2541637677 ecr 2962246962], length 0
+    01:36:24.608867 IP 10.16.215.55.60876 > 10.16.70.187.80: Flags [P.], seq 1:96, ack 1, win 221, options [nop,nop,TS val 2541637677 ecr 2962246962], length 95: HTTP: HEAD /index.html HTTP/1.1
+    01:36:24.608876 IP 10.16.70.187.80 > 10.16.215.55.60876: Flags [.], ack 96, win 219, options [nop,nop,TS val 2962246962 ecr 2541637677], length 0
+    01:36:24.609007 IP 10.16.70.187.80 > 10.16.215.55.60876: Flags [P.], seq 1:239, ack 96, win 219, options [nop,nop,TS val 2962246962 ecr 2541637677], length 238: HTTP: HTTP/1.1 200 OK
+    01:36:24.609052 IP 10.16.215.55.60876 > 10.16.70.187.80: Flags [.], ack 239, win 229, options [nop,nop,TS val 2541637677 ecr 2962246962], length 0
+
+ServiceMatcher
+---------------
+
+This type of configuration is specified using Kubernetes service name and namespace
+for which traffic needs to be redirected.
+The example shows how to redirect from traffic matching ``my-service``, to a
+backend pod deployed with labels ``app=proxy`` and Layer 4 port ``80``
+with protocol ``TCP``.
+
+Create the custom resource of type CiliumLocalRedirectPolicy with ``serviceMatcher``
+configuration.
+
+.. literalinclude:: ../../examples/kubernetes-local-redirect/lrp-svcmatcher.yaml
+
+.. code-block:: bash
+
+    $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes-local-redirect/lrp-svcmatcher.yaml
+
+Verify that the custom resource is created.
+
+.. code-block:: bash
+
+    $ kubectl get ciliumlocalredirectpolicies | grep svc
+    NAME               AGE
+    lrp-svc   20h
+
+Deploy the Kubernetes service for which traffic needs to be redirected.
+
+.. literalinclude:: ../../examples/kubernetes-local-redirect/k8s-svc.yaml
+
+.. code-block:: bash
+
+    $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes-local-redirect/k8s-svc.yaml
+
+Verify that the service is created.
+
+.. code-block:: bash
+
+    $ kubectl get service | grep 'my-service'
+    my-service   ClusterIP   172.20.0.51   <none>        80/TCP     2d7h
+
+Deploy a backend pod where the traffic needs to be redirected to. The metadata
+labels and container port and protocol match with the labels, port and protocol
+fields specified in the CiliumLocalRedirectPolicy custom resource created
+above, respectively.
+
+.. literalinclude:: ../../examples/kubernetes-local-redirect/backend-pod.yaml
+
+.. code-block:: bash
+
+    $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes-local-redirect/backend-pod.yaml
+
+Verify that the pod is running.
+
+.. code-block:: bash
+
+    $ kubectl get pods | grep lrp-pod
+    lrp-pod                      1/1     Running   0          46s
+
+Deploy a client pod that will invoke a curl command to the Cluster IP of
+``my-service`` specified in the ``lrp-svc`` custom resource above.
```

Comment:
I wonder if it's better to consolidate the backend / client pod creation towards the start of the doc after Cilium pod creation so it doesn't have to be duplicated for each section?